lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -477,6 +477,9 @@ SILPassPipelinePlan::getPerformancePassPipeline(const SILOptions &Options) {
   // stdlib.
   addPerfEarlyModulePassPipeline(P);
 
+  // Cleanup after SILGen: remove trivial copies to temporaries.
+  P.addTempRValueOpt();
```

Comment:
ok. If I have to make some other changes I'll do it. Otherwise in a follow-up commit